automation/clients/secrets.py

```diff
@@ -0,0 +1,32 @@
+from google.cloud import secretmanager
+
+from .. import config
+
+##########
+# CLIENT #
+##########
+
+
+class SecretsClient:
+    def __init__(self):
+        self._client = secretmanager.SecretManagerServiceClient()
+        self._project_id = config.load().google_cloud.project_id
```

Comment:
idk what's better, getting the config again at this point in the runtime or having to pass the Google Cloud config into the `Auth0Client` initializer

Reply:
I'd prefer passing the config in to the constructor.